### DISCUSSION TITLE
Ram/Keyboard on b450 Tomohawk Max

### DISCUSSION THREAD

**Discussion by FreckledSea21** [ixjf/MSIRGB#240] [Q&A]
Hey guys,

Just wanted to let you know my T Force DELTA RGB DDR4 GAMING MEMORY 3200 and Ripjaw RGB Keyboard both arent recognised by the program. Is this just a limitation of the software or a bug?

Otherwise fantastic!

**Comment by ixjf** [comment]
Thanks. MSIRGB only controls motherboards.
